hack/run-functests.sh

```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 set -x
 which ginkgo
+ginkgo version
 if [ $? -ne 0 ]; then
 # we are moving to a temp folder as in go.mod we have a dependency that is not
 # resolved if we are not using google's GOPROXY. That is not the case when building as
 # we are using vendored dependencies
 	GINKGO_TMP_DIR=$(mktemp -d)
 	cd $GINKGO_TMP_DIR
 	go mod init tmp
-	GOFLAGS=-mod=mod go install github.com/onsi/ginkgo/ginkgo@v1.12.0
+	GOFLAGS=-mod=mod go install github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
Why latest, rather than hardcoding to a version?

Reply:
This way we can enforce that we use the lastest ginkgo with better features and functionalities. If there is any package incompatible to the latest version of ginkgo, we are forced to update that too.

Reply:
chances of breaking is high with latest . Please pin to a stable version